help/sources/connectors/marketing-automation/chatlio-webhook.md

```diff
@@ -0,0 +1,49 @@
+---
+title: chatlio Source Overview
+description: Learn how to connect chatlio to Adobe Experience Platform using APIs or the user interface by leveraging webhooks 
+---
+# (Beta) [!DNL Chatlio]
+
+>[!NOTE]
+>
+>The [!DNL Chatlio] source is in beta. See the [sources overview](../../home.md#terms-and-conditions) for more information on using beta-labelled sources.
+
+Adobe Experience Platform allows data to be ingested from external sources while providing you with the ability to structure, label, and enhance incoming data using Platform services. You can ingest data from a variety of sources such as Adobe applications, cloud-based storage, databases, and many others.
+
+Experience Platform provides support for ingesting data from Streaming applications. Support for streaming providers include [!DNL Chatlio].
+
+[[!DNL Chatlio]](https://chatlio.com/) is a live chat app that is fully integrated with Slack and facilitates multiple support agents to help an individual site visitor simultaneously. Using the [!DNL Zapier App](https://chatlio.com/docs/zapier/) you can connect [!DNL Chatlio] to over 2000 different apps and services.
+
+The [!DNL Chatlio] source allows you to ingest supported Webhook Event Schemas and their associated event data from [!DNL Chatlio.com] using the [[!DNL Chatlio] Webhooks](https://chatlio.com/docs/webhooks/).
+
+The supported webhooks are:
+* Exporting chat transcripts
+* Exporting offline messages
+* New conversation has started
+* Visitor did not get an answer in time
+* Visitor left feedback after a chat
+
+## Prerequisites {#prerequisites}
+
+Before you can create a [!DNL Chatlio] source connection, you must first ensure that you have the following:
+
+A validated [!DNL Chatlio] account. If you do not have one already see the [[!DNL Chatlio] signup page](https://chatlio.com/app/#/signup) to register and create your account.
```

Comment:
Let's revise this one to either: "A valid [!DNL Chatlio] account." or "A [!DNL Chatlio] account."